licenses.go

```diff
@@ -41,30 +41,44 @@ func SetLicenseFileForDependency(license_file string) {
 	known_license_files = append(known_license_files, "--add-license="+license_file)
 }
 
+var firstEnterpriseCommit = ""
+
+// This should be set to the oldest commit that is not part of Open Source, only
+// part of Enterprise, if any. IOW it should be the very first commit after the
+// fork point, on the Enterprise branch.
+func SetFirstEnterpriseCommit(sha string) {
+	firstEnterpriseCommit = sha
+}
+
 func CheckLicenses(t TSubset) {
```

Comment:
As we will need to update our repos to update they vendored version of mendertesting... If it is not too much work we can split this test into `CheckLicenses` and `CheckCommits`.

Alternatively, just rename the method to `CheckMenderCompliance` or something like that which hints that there are more thing than license checks.

Reply:
Alright, good point. I did the latter.